### PR TITLE
Remove deprecated features

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

File: Doctrine/Phpcr/StaticContent.php [hunk 1]
```diff
@@ -11,14 +11,13 @@
...
 namespace Symfony\Cmf\Bundle\ContentBundle\Doctrine\Phpcr;
 
-use Doctrine\ODM\PHPCR\HierarchyInterface;
 use PHPCR\NodeInterface;
 use Symfony\Cmf\Bundle\ContentBundle\Model\StaticContent as ModelStaticContent;
 
 /**
  * PHPCR specific static content.
  */
-class StaticContent extends ModelStaticContent implements HierarchyInterface
```
Comment (dbu) [inline Doctrine/Phpcr/StaticContent.php]: hm. the HierarchyInterface also has getParent / setParent deprecated. ideally we would synchronize the removal there to keep the interface on the content.

Comment (WouterJ) [inline Doctrine/Phpcr/StaticContent.php]: Yeah, problem is that we then should release Doctrine PHPCR-ODM 2.0. Or you should simply remove the deprecated methods from the interface, don't see anything wrong with that.

Comment (dbu) [inline Doctrine/Phpcr/StaticContent.php]: i created a 2.0 branch for the odm, see
https://github.com/doctrine/phpcr-odm/pull/711

if i understand composer correctly, you can simply depend on phpcr-odm
`2.0@dev` to make this work. though there are likely conflicts then with
other dependencies?

i'd like to keep this as a 2.0 branch for now, until we get closer to
release the cmf 2.0 versions and then switch phpcr-odm over to 2.0

Comment (WouterJ) [inline Doctrine/Phpcr/StaticContent.php]: > though there are likely conflicts then with other dependencies?

Yeah, we'll need to go full blown dev stability now in all CMF packages (as we need to require SonataDoctrinePhpcrAdminBundle 2.0-dev)

**Pull request merged**